streams/src/main/java/org/apache/kafka/streams/processor/internals/StreamThread.java

```diff
@@ -384,9 +390,8 @@ public static StreamThread create(final InternalTopologyBuilder builder,
             builder,
             threadId,
             logContext,
-            assignmentErrorCode,
```

Comment:
this diff is a bit off. The assignmentErrorCode was added in this PR. I didn't change the order of the params for no reason...